app/src/main/java/com/example/parkingsystem/activity/ParkingActivity.java

```diff
@@ -23,6 +23,7 @@ protected void onCreate(Bundle savedInstanceState) {
         presenter = new ParkingPresenter(new ParkingModel(), new ParkingView(this, binding));
 
         binding.buttonMainSetAmountParkingSpaces.setOnClickListener(view -> presenter.inflateDialog(this));
+        binding.buttonMainReservation.setOnClickListener(view -> presenter.moveToOtherActivity());
```

Comment:
moveToOtherActivity is a very generic name, you can use something more like onReservationButtonClicked for example

Reply:
Changed